smart_open/tests/test_hdfs.py

```diff
@@ -31,23 +32,29 @@
 # such as Windows.
 #
 class CliRawInputBaseTest(unittest.TestCase):
+    def _setUp(self, test_file):
+        if platform == "linux" or platform == "linux2":
+            path = P.join(CURR_DIR, 'test_data/' + test_file)
```

Comment:
The join function will handle slashes for you (also same thing below).

```suggestion
            path = P.join(CURR_DIR, 'test_data' + test_file)
```